client/package.json

```diff
@@ -133,13 +135,16 @@
     "@babel/types": "7.20.7",
     "@codesee/babel-plugin-instrument": "0.470.0",
     "@codesee/tracker": "0.470.0",
+    "@faker-js/faker": "7.6.0",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "12.1.5",
+    "@types/faker": "6.6.9",
     "autoprefixer": "10.4.13",
     "babel-plugin-transform-imports": "2.0.0",
     "chokidar": "3.5.3",
     "copy-webpack-plugin": "9.1.0",
     "core-js": "2.6.12",
+    "faker": "6.6.6",
```

Comment:
We're using `@faker-js/faker` - are we sure we need `faker` too?

Additionally, `@faker-js/faker` uses TypeScript so we shouldn't need `@types/faker`?

Reply:
_Can we fake using faker so that we use faker by faking it?_

Reply:
@naomi-lgbt nah, that was just a mistake.  I forgot that the original faker repo had been replaced with nonsense and I somehow forgot to remove it :face_exhaling:

Reply:
I am removing faker from the client in the Analytics migration pr, so we could leave it off here.

Reply:
Is this okay to merge or are we waiting for the analytics pr?